subsys/bluetooth/host/hci_core.c

```diff
@@ -1677,35 +1677,10 @@ static int set_flow_control(void)
 }
 #endif /* CONFIG_BT_HCI_ACL_FLOW_CONTROL */
 
-static int bt_clear_all_pairings(u8_t id)
-{
-	bt_conn_disconnect_all(id);
-
-	if (IS_ENABLED(CONFIG_BT_SMP)) {
-		bt_keys_clear_all(id);
-	}
-
-	if (IS_ENABLED(CONFIG_BT_BREDR)) {
-		bt_keys_link_key_clear_addr(NULL);
-	}
-
-	return 0;
-}
-
-int bt_unpair(u8_t id, const bt_addr_le_t *addr)
+static void _bt_unpair(u8_t id, const bt_addr_le_t *addr)
```

Comment:
We don't normally use _bt for internal function, normally we just drop the prefix. So I would suggest to rename this to just `unpair`